c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -68,6 +90,47 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: template});
 };
 
+
+/**
+ * @param {Date} dateStart
+ * @param {Date} dateEnd
+ * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
```

Comment:
Do your changes fix the "suppress" problem? Including in ``documentediting.js``?
As said in an other comment, we might not need function ``displayDate`` so it's not worth spending time fixing it, but would be cool for the other.